packages/message-list/all-imports.js

```diff
@@ -1,6 +1,4 @@
 import './vaadin-message.js';
 import './vaadin-message-list.js';
-import './vaadin-message-input.js';
```

Comment:
We don't need this file in the new package. Having `vaadin-message-list.js` as an entrypoint is enough.
Please follow the same approach as for example `vaadin-tabs` package.